### ISSUE TITLE
rename consumers to be delivery_points and producers to be receipt_points

### ISSUE THREAD

**Issue by rb004f** [lanl-ansi/GasModels.jl#122]
This naming convention is consistent with the gas industry naming conventions.  This will be a pretty big breaking change on the input format, so may require some discussion before doing this transition.

**Comment by ccoffrin** [comment]
How about `delivery` and `receipt`?  I think that would be more consistent with the naming in other sectors.

Note we are standardizing around singular for component lists, so it would be `delivery_point` and `receipt_point`.

**Comment by rb004f** [comment]
Yes, I think that would work. @kaarthiksundar when you are working on finalizing the matlab format, lets call the consumer and producer tables receipt and delivery. Internally, in the io functions, we can change the name to consumer and producer until we make the internal change....
